types/three/three-core.d.ts

```diff
@@ -2747,7 +2747,10 @@ export interface LineDashedMaterialParameters extends MaterialParameters {
     gapSize?: number;
 }
 
-export class LineDashedMaterial extends Material {
+/**
+ * Changed LineDashedMaterial to extend LineBasicMaterial as per the source
+ */
+export class LineDashedMaterial extends LineBasicMaterial {
     constructor(parameters?: LineDashedMaterialParameters);
 
     color: Color;
```

Comment:
Can we add a definition for the `isLineDashedMaterial` property?

Reply:
See here: https://github.com/mrdoob/three.js/blob/2410ad1a8f5844df00191600eafda59def537830/src/materials/LineDashedMaterial.js#L35

Reply:
Hi Daniel, thanks, added it. Is there anything else you would want me to add?